status/reporter_gitlab_test.go

```diff
@@ -77,6 +77,7 @@ var _ = Describe("GitLabReporter", func() {
 					"build.appstudio.redhat.com/commit_sha":             digest,
 					"appstudio.redhat.com/updateComponentOnSuccess":     "false",
 					"pac.test.appstudio.openshift.io/repo-url":          repoUrl,
+					"pac.test.appstudio.openshift.io/target-project-id": projectID,
```

Comment:
because we used the same ID for source and target, we are not really testing in unittests that the right ID is used for commit status vs for comment. It should be different and test mock updated accordingly to expect desired ID

Reply:
Fixed. Check gitlab api function with correct project id is called. Thanks.